html/pages/device/showconfig.inc.php

```diff
@@ -318,3 +333,28 @@
 }//end if
 
 $pagetitle[] = 'Config';
+
+echo '
+<script type="text/javascript">
+    function refresh_oxidized_node(device_hostname){
```

Comment:
This function already exists in `html/pages/oxidized.inc.php`. You will need to move it to `html/js/librenms.js` rather than duplicate it.